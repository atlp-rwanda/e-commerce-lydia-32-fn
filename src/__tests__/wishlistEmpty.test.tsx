// src/__tests__/wishlistEmpty.test.tsx

import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import WishlistEmpty from '../Components/wishlistEmpty';

describe('WishlistEmpty Component', () => {
  it('should render the WishlistEmpty component', () => {
    render(<WishlistEmpty />);

    expect(screen.getByText('WISHLIST')).toBeInTheDocument();
    expect(screen.getByText('No Items in your wishlist')).toBeInTheDocument();
  });
});
