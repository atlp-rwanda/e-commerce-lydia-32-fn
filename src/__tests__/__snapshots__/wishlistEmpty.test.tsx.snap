// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`WishlistEmpty Component > should match the snapshot 1`] = `
<DocumentFragment>
  <section
    class="text-center flex flex-col justify-center items-center h-96 bg-gray-50 mt-16"
  >
    <div
      class="text-5xl font-semibold tracking-widest"
    >
      WISHLIST
    </div>
    <div
      class="text-lg text-gray-400 tracking-tight font-medium"
    >
      No Items in your wishlist
    </div>
  </section>
</DocumentFragment>
`;
